grubby.8

```diff
@@ -1,42 +1,61 @@
-       [--bad-image-okay] [--boot-filesystem=\fIbootfs\fR] 
-       [--bootloader-probe] [--config-file \fIpath\fR] [--copy-default]
-       [--debug] [--default-kernel] [--default-index] [--default-title]
-       [--devtree=\fIdevicetree.dtb\fR] [--set-index=\fIentry-index\fR]
-       [--grub] [--lilo] [--yaboot] [--silo] [--zipl]
-       [--info=\fIkernel-path\fR] [--initrd=\fIinitrd-path\fR] 
-       [--make-default] [-o path] [--version]
-       [--remove-kernel=\fIkernel-path\fR] [--remove-args=\fIargs\fR]
-       [--set-default=\fIkernel-path\fR] [--set-default-index=\fIentry-index\fR]
-       [--title=entry-title] [--add-multiboot=\fImultiboot-path\fR]
-       [--mbargs=\fIargs\fR] [--remove-multiboot=\fImultiboot-path\fR]
-       [--remove-mbargs=\fIargs\fR]
+\fBgrubby\fR [\fIOPTIONS...\fR]
 
 .SH DESCRIPTION
 \fBgrubby\fR is a command line tool for updating and displaying information
-about the configuration files for the \fBgrub\fR, \fBlilo\fR, \fBelilo\fR
-(ia64), \fByaboot\fR (powerpc) and \fBzipl\fR (s390) boot loaders. It
-is primarily designed to be used from scripts which install new
-kernels and need to find information about the current boot environment.
-
-On BIOS-based Intel x86 platforms, \fBgrub2\fR is the default bootloader and
-the configuration file is in \fB/boot/grub2/grub.cfg\fR.  On UEFI-based Intel
-x86 platforms, \fBgrub2\fR is the default bootloader, and the configuration
-file is in \fB/boot/efi/EFI/redhat/grub.cfg\fR.  On Intel ia64 platforms,
-\fBelilo\fR mode is used and the default location for the configuration file
-is \fB/boot/efi/EFI/redhat/elilo.conf\fR. On PowerPC platforms, systems based
-on Power8 now support \fBgrub2\fR as a bootloader and store using a default
-config stored in \fB/boot/grub2/grub.cfg\fR. The earlier Power7 systems use \fByaboot\fR
-parsing and the configuration file should be in \fB/etc/yaboot.conf\fR.  On
-s390 platforms the \fBzipl bootloader\fR will read from \fB/etc/zipl.conf\fR.
-
+about the configuration files for various architecture specific bootloaders.
+It is primarily designed to be used from scripts which install new kernels
+and need to find information about the current boot environment.
+
+.SS Architecture Support
+The \fBgrubby\fR executable has full support for the \fBgrub2\fR
+bootloader on \fBx86_64\fR systems using legacy BIOS or modern
+UEFI firmware and \fBppc64\fR and \fBppc64le\fR hardware using
+OPAL or SLOF as firmware.
+
+Legacy \fBs390\fR and the current \fBs390x\fR architectures
+and their \fBzipl\fR bootloader are fully supported.
+
+Support for \fByaboot\fR has been deprecated as all ppc architecture
+hardware since the Power8 system uses \fBgrub2\fR or petitboot
+which both use the grub2 configuration file format.
+
+Legacy bootloaders \fBLILO\fR, \fBSILO\fR, and \fBELILO\fR
+are deprecated in favor of previously mentioned bootloaders. The
+\fBSILO\fR bootloader should also be considered unsupported.
```

Comment:
This last SILO sentence feels a bit too harsh.  Really, what I was wanting to convey is that support for configuring SILO via grubby should be considered deprecated and effectively unsupported.  In fact, any of the boot loaders that grubby supports that we don't actively use anymore should be considered deprecated and unsupported within the context of grubby.  Does grubby have support for other bootloaders, like aboot or MILO?  There could be a statement that classifies support for these boot loaders in grubby as deprecated and unsupported.